src/Chapter2.hs

```diff
@@ -335,8 +358,13 @@ from it!
 
 ghci> :l src/Chapter2.hs
 -}
+
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList _ _ [] = []
+subList start finish (x:xs)
+  | start < 0 || finish < 0 = []
+  | start > 0 = subList (start - 1) (finish - 1) xs
+  | otherwise = x : subList start (finish - 1) xs
```

Comment:
Oh, interesting recursive solution 💡 

You also can use `take` and `drop` functions here 🙂